docs/HowTo/Migrate-Database.md

```diff
@@ -5,17 +5,78 @@ description: How to migrate a RocksDB database to a LevelDB2 database.
 # Migrate the database
 
 As of v21.5.0, new Teku installations use a LevelDB2 database. Previous Teku versions use
-RocksDB which continue to be supported.
+RocksDB which continues to be supported.
 
-LevelDB2 databases use less memory and are proven to be more stable with Teku. Users can migrate
-Teku nodes running a RocksDB database to a LevelDB2 database if the Teku node runs in [pruning mode].
+LevelDB2 databases use less memory and proves to be more stable with Teku.
+
+!!! important
+
+    We recommend Teku nodes running in [archive mode] use the [automatic migration](#automatic-migration) method, and
+    Teku nodes running in [prune mode] use the manual migration.
+
+## Automatic migration
+
+Teku nodes running in [archive mode] must resynchronize from genesis to migrate from RocksDB to LevelDB2. Use the
+[`migrate-database`](../Reference/CLI/Subcommands/Migrate-Database.md) subcommand to migrate the database.
+Once migration completes there will be two databases in Teku's data folder. Manually verify that Teku starts
+and has the expected data before you remove the RocksDB database.
+
+!!! note
+
+    Disk space requirements may be as much as the existing storage folder used by Teku. This storage
+    requirement will only be for the period of the `migrate-database` command, plus long enough to validate that
+    the new database is working.
```

Comment:
I might have worded this badly...

So the rocksdb storage requirement might be 200GB, then while we're migrating, we might use an extra 200GB, but once you're done, you can delete the 200GB from rocks, and be back to 1x storage amt...